lib/compare-file-size.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const promisify = require('bluebird').promisify;
+const glob = promisify(require('glob').glob);
+const getFileSizesPerType = require('./get-filesizes-per-type');
+
+function compareFileSize(sourceFolder, tempFolder) {
```

Comment:
this module doesn't need to know about `temp`, and to stick with other vocabulary I would use `(srcDir, destDir)`.